handlers/consumer.go

```diff
@@ -0,0 +1,119 @@
+package handlers
+
+import (
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	"github.com/memphisdev/memphis.go"
+	"rest-gateway/logger"
+	"rest-gateway/models"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type requestBody struct {
+	ConsumerName      string `json:"consumer_name"`
+	ConsumerGroupName string `json:"consumer_group_name"`
+	BatchSize         int    `json:"batch_size"`
+	MaxAckTime        int    `json:"max_ack_time"`
+	BatchMaxWaitTime  int    `json:"batch_max_wait_time"`
+	MaxMsgDeliveries  int    `json:"max_msg_deliveries"`
```

Comment:
remove this one

Reply:
and you missed 
startConsumeFromSequence and lastMessages that should also be supported

Reply:
I couldn't find the two methods `FetchStartConsumeFromSeq` and `FetchLastMessages` in the SDK
![image](https://github.com/memphisdev/memphis-rest-gateway/assets/61353651/a24da1b5-1787-4022-b135-bf6a03dc03e5)